site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -575,6 +577,36 @@ components:
                      - all
                      - none
 
+      DataVolume:
+         type: object
+         additionalProperties: false
+         properties:
+            "effective-since":
```

Comment:
this property seems to be mandatory, we should also mark it as such in the value object ...